openlibrary/templates/lib/nav_foot.html

```diff
@@ -7,6 +7,7 @@
         <h2>Open Library</h2>
         <ul>
           <li><a href="/about/vision" title="$_('Vision')">$_('Vision')</a></li>
+          <li><a href="https://openlibrary.org/volunteer" title="$_('Volunteer')">$_('Volunteer')</a></li>
```

Comment:
No need for the domain (future note: we should remove all the duplicate `title` text; it's bad practice. But not related to your changes, and it's better to be consistent :) )

```suggestion
          <li><a href="/volunteer" title="$_('Volunteer')">$_('Volunteer')</a></li>
```